modules/reindex/licenses/httpcore-4.4.4.jar.sha1

```diff
@@ -0,0 +1 @@
+b31526a230871fbe285fbcbe2813f9c0839ae9b0
```

Comment:
I know this might not be your dependency (more from the http client), but we should probably bump to 4.4.5: https://issues.apache.org/jira/browse/HTTPCORE/fixforversion/12334619/?selectedTab=com.atlassian.jira.jira-projects-plugin:version-summary-panel